lib/addons/ebs-csi-driver/index.ts

```diff
@@ -1,26 +1,42 @@
-import { CoreAddOn } from "../core-addon";
-import { getEbsDriverPolicyDocument } from "./iam-policy";
-
-/**
- * Default values for the add-on
- */
-const defaultProps = {
-    addOnName: 'aws-ebs-csi-driver',
-    version: 'v1.14.0-eksbuild.1',
-    saName: 'ebs-csi-controller-sa'
-};
-
-/**
- * Implementation of EBS CSI Driver EKS add-on.
- */
-export class EbsCsiDriverAddOn extends CoreAddOn {
-
-    constructor(version?: string) {
-        super({
-            addOnName: defaultProps.addOnName,
-            version: version ?? defaultProps.version,
-            saName: defaultProps.saName,
-            policyDocumentProvider: getEbsDriverPolicyDocument
-        });
-    }
-}
\ No newline at end of file
+import { CoreAddOn } from "../core-addon";
+import { getEbsDriverPolicyDocument } from "./iam-policy";
+
+/**
+ * Interface for EBS CSI Driver EKS add-on options
+ */
+interface IEbsCsiDriverAddOnOptions {
+  /**
+   * Version of the driver to deploy
+   */
+  version?: string;
+  /**
+   * List of KMS keys to be used for encryption
+   */
+  kmsKeys?: string[];

```

Comment:
@shapirov103 do we really need a list of KMS keys? Don't we need just one to encrypt an EBS volume?

Reply:
We do need a list, because the cluster may have many teams and each team may require its own encryption key. These keys are only fed to the IAM policy here, so I see no reason to restrict it to just one. And the actual tightening of resource access policies to particular keys (as opposed to wildcard) was recommendation from Steve.